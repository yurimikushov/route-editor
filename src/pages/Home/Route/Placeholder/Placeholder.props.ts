type PlaceholderProps = {
  className?: string
}

export default PlaceholderProps
