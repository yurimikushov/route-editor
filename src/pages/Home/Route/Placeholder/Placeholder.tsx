import { FC } from 'react'
import cn from 'classnames'
import PlaceholderProps from './Placeholder.props'

const Placeholder: FC<PlaceholderProps> = ({ className }) => {
  return (
    <div
      className={cn(
        className,
        'w-full h-24 p-3',
        'flex justify-center items-center',
        'text-sm text-gray-500'
      )}
    >
      Маршрут пока пустой
    </div>
  )
}

export default Placeholder
