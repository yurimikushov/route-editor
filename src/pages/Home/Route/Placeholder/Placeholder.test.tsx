import renderer from 'react-test-renderer'
import Placeholder from './Placeholder'

it('should render placeholder', () => {
  const tree = renderer
    .create(<Placeholder className='empty-placeholder' />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
