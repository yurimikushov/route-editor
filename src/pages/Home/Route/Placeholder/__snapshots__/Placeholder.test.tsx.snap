// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render placeholder 1`] = `
<div
  className="empty-placeholder w-full h-24 p-3 flex justify-center items-center text-sm text-gray-500"
>
  Маршрут пока пустой
</div>
`;
